.github/workflows/industrial_ci_action.yaml

```
@@ -0,0 +1,26 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+# This determines when this workflow is run
+on:
+  # When kinetic-devel branch is pushed to
+  push:
+    branches: [ kinetic-devel ]
+  # When there is a pull request against kinetic-devel
+  pull_request:
+    branches: [ kinetic-devel ]
```

I guess this can be simplified: On other branches, the workflow isn't defined at all (or differently).
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events
```suggestion
on: [push, pull_request]
```